src/logger/loggingConstants.js

```diff
@@ -0,0 +1,13 @@
+// Activity Sidebar
+export const ACTIVITY_SIDEBAR_TAGS = {
+    Initialized: 'activity_sidebar_init',
+    JSReady: 'activity_sidebar_js',
+};
+
+// Activity Feed
+export const ACTIVITY_FEED_TAGS = {
```

Comment:
maybe rename to constants.js since its already namespaced by the folder structure